newIDE/app/src/ProjectManager/index.js

```diff
@@ -826,14 +833,14 @@ export default class ProjectManager extends React.Component<Props, State> {
                       onMoveUp={() => this._moveUpLayout(i)}
                       canMoveDown={i !== project.getLayoutsCount() - 1}
                       onMoveDown={() => this._moveDownLayout(i)}
-                      buildExtraMenuTemplate={() => [
+                      buildExtraMenuTemplate={(i18n: I18nType) => [
                         {
-                          label: 'Edit Scene Properties',
+                          label: i18n._(t`Edit Scene Properties`),
                           enabled: true,
                           click: () => this._onOpenLayoutProperties(layout),
                         },
                         {
-                          label: 'Edit Scene Variables',
+                          label: i18n._(t`Edit Scene Variables`),
                           enabled: true,
                           click: () => this._onOpenLayoutVariables(layout),
                         },
```

Comment:
These labels don't work, help is needed, otherwise the other work fine, flow is totally broken the scan fail often i hope  i didn't miss anything else.

Reply:
Found :)

`import { I18n } from '@lingui/react';` was required in ContextMenu.js

Reply:
Ok :) I've seen that you passed i18n as argument to buildExtraMenuTemplate. Seems like a good idea I think. Would be interesting to follow this convention everywhere.